Formula/ssreflect.rb

```diff
@@ -36,7 +76,27 @@ def install
       system "make", "-f", "Makefile.coq", "install-doc", *args
     end
     bin.install "bin/ssrcoq.byte", "bin/ssrcoq" if build.with? "static"
-    (share/"ssreflect").install "pg-ssr.el"
+    pkgshare.install "pg-ssr.el"
+  end
+
+  test do
+    (testpath/"helloworld.v").write <<-EOS.undent
+      Add LoadPath "#{lib}/coq/user-contrib/Ssreflect" as Ssreflect.
+      Require Import Ssreflect.ssreflect.
+      Variable P:Prop.
+
+      Theorem helloworld: P -> P.
+      Proof.
+        done.
+      Qed.
+
+      Check helloworld.
+    EOS
+    (testpath/"expected").write <<-EOS.undent
+      helloworld
+           : P -> P
+    EOS
+    assert_equal File.read(testpath/"expected"), pipe_output("ssrcoq -compile helloworld")
```

Comment:
If `ssrcoq` is part of this package this should be `("#{bin}/ssrcoq -compile helloworld")` though 😉.